### PR TITLE
Fire operational webhook on endpoint secret rotation

### PR DESCRIPTION
**Pull request by svix-andor**
As per title, rotating the secrets of an endpoint is an update to the endpoint, therefore it should fire the corresponding operational webhook.

### CODE REVIEW

File: server/svix-server/src/v1/endpoints/endpoint/mod.rs [hunk 1]
```diff
@@ -374,7 +374,7 @@ impl From<(endpoint::Model, Metadata)> for EndpointOut {
     }
 }
 
-#[derive(Clone, Debug, PartialEq, Eq, Validate, Deserialize, JsonSchema)]
+#[derive(Default, Clone, Debug, PartialEq, Eq, Validate, Serialize, Deserialize, JsonSchema)]
```
Comment (tasn) [inline server/svix-server/src/v1/endpoints/endpoint/mod.rs:377]: Why is this needed?

Comment (tasn) [reply server/svix-server/src/v1/endpoints/endpoint/mod.rs:377]: Ah, nvm, just for the tests. :)

**Review by svix-gabriel — APPROVED** [review]
